whats-new-in-pgroll-0-6-0.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: pgroll 0.6.0 update
+description: Learn about the latest changes in pgroll
```

Comment:
```suggestion
description: Learn about the latest changes in pgroll since its 0.1.0 release as we continue to build and turn it into a first-class open-source schema migration tool for Postgres.
```

Reply:
I changed this to:

```
Learn about the latest changes in pgroll in the 0.6.0 release as we 
continue to build and turn it into a first-class open-source schema
migration tool for Postgres.
```

as the blog is really only talking about changes in `0.6.0`, not everything since `0.1.0`.